models/post.py

```diff
@@ -30,5 +33,8 @@ def __str__(self):
         """
 
         group = self.group
-        text = self.text[:80]
-        return f'{group}: {text}'
+        text = self.text[:80]        
+        clean = re.compile('<.*?>')
```

Comment:
Why do you need to create a RegEx object? [`re.sub`](https://docs.python.org/3/library/re.html#re.sub) accepts a RegEx as a string too. Directly use `re.sub(r'<.*?>', '', text)`